File: java8/src/test/java/com/shekhargulati/ninetynine_problems/_02_arithmetic/P37Test.java
package com.shekhargulati.ninetynine_problems._02_arithmetic;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.equalTo;
import static org.junit.Assert.assertThat;

public class P37Test {

    @Test
    public void gcdOf36And63Is9() throws Exception {
        int gcd = P37.gcd(36, 63);
        assertThat(gcd, equalTo(9));
    }
}

File: scala/src/test/scala/com/shekhargulati/ninetynine_problems/_02_arithmetic/P37Test.scala
package com.shekhargulati.ninetynine_problems._02_arithmetic

import org.scalatest.{FunSpec, Matchers}

class P37Test extends FunSpec with Matchers {

  describe("P37 Spec") {
    it("greatest common divisior of 36 and 63 is 9") {
      val result = P37.gcd(36, 63)
      result should be(9)
    }
  }

}